### PR TITLE
hw-probe: new, 1.6.5

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

hw-probe: new, 1.6.5

Package(s) Affected
-------------------

- hw-probe

Security Update?
----------------

No

Test Build(s) Done
------------------

- [x] noarch

Update(s) Uploaded to Stable
----------------------------

- [x] noarch

### CODE REVIEW

File: app-utils/hw-probe/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PKGNAME=hw-probe
+PKGSEC=utils
+PKGDEP="perl hwinfo curl dmidecode smartmontools pciutils usbutils edid-decode"
+PKGDES="Probe for hardware, check operability and find drivers"
```
Comment (MingcongBai) [inline app-utils/hw-probe/autobuild/defines]: A tool to probe, test, and find drivers for your hardware

File: app-utils/hw-probe/spec [hunk 2]
```diff
@@ -0,0 +1,3 @@
+VER=1.6.5
+SRCS="tbl::https://github.com/linuxhw/hw-probe/archive/refs/tags/$VER.tar.gz"
+CHKSUMS="sha256::42030ba2fb3f6fb0772ab34744fbb91a89b1b6a9b0ed99e861fa05ff86968fb1"
```
Comment (MingcongBai) [inline app-utils/hw-probe/spec:3]: Missing CHKUPDATE.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/hw-probe/spec [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+CHKUPDATE="github::repo=linuxhw/hw-probe"
```
Comment (MingcongBai) [inline app-utils/hw-probe/spec]: Please use Anitya, I have registered an ID for you - 368914.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

File: app-utils/hw-probe/spec (hunk 3 above)
Comment (jiegec) [reply app-utils/hw-probe/spec]: done

File: app-utils/hw-probe/autobuild/defines (hunk 1 above)
Comment (jiegec) [reply app-utils/hw-probe/autobuild/defines]: done

**Pull request merged**